src/openforms/emails/templates/emails/cosign.html

```diff
@@ -0,0 +1,25 @@
+{% load static i18n %}
+
+<p>
+    {% blocktranslate %}
+        This is a request to co-sign form "{{ form_name }}".<br />
+    {% endblocktranslate %}
+    
+    {% if show_form_link %}
+        {% blocktranslate %}
+            You can go to the form page also by clicking on:<br />
+            <a class="utrecht-link utrecht-link--openforms" href="{{ form_url }}">this link</a><br />
+        {% endblocktranslate %}
```

Comment:
```suggestion
        {% blocktranslate trimmed %}
            Please visit the form page by navigating to the following link: {{ form_url|urlize }}.
        {% endblocktranslate %}
```
`urlize` ensures that it's a clickable link. The CSS class names won't have an effect in e-mail templates, so there's not much point in using these.

Keep in mind that plain text versions of HTML emails are also automatically created if not explicitly provided, and those strip out the HTML tags, so this variant still has the URL in the body for users to copy and paste in their browser if they use the plain text email.